hpedockerplugin/volume_manager.py

```diff
@@ -372,6 +372,12 @@ def manage_existing(self, volname, existing_ref, backend='DEFAULT',
             LOG.exception(msg)
             return json.dumps({u"Err": six.text_type(msg)})
 
+        if ('rcopyStatus' in existing_ref_details and
```

Comment:
We can now check whether the volume is associated with the remote copy group. If yes, we do not allow importing the volume on docker plugin

Reply:
Looks okay